simulator/src/test/java/com/hazelcast/simulator/coordinator/RunTestSuiteTaskTest.java

```diff
@@ -470,4 +472,32 @@ public void run() {
             }
         }
     }
+
+//    @Test(timeout = 10000)
```

Comment:
Please use `@Ignore` to disable tests. It's not easy to find disabled tests like this. It's also not visible during a test execution that there are tests, which need to be fixed (ignored tests are displayed).